### PR TITLE
fix(DB/creature): Improve AQ40 Bug Tunnel

### PR DESCRIPTION
**Pull request by Gultask**
## Changes Proposed:
-  Link Vekniss Drone spawns that are contained in a formation to Fankriss
-  Link Vekniss Soldier spawns to Fankriss
-  Creates linked formations to Vekniss Drones packs
-  Adjusts wander_distance to match a 20-30 minute sniff
-  Adjusts respawn timers for Drones and Soldiers
-  Adds SAI to Drones to force despawn after 5s

## Issues Addressed:

- Closes unopened issue

## SOURCE:

- Retail for wander_distance, formation and linking
- Vmangos for respawn timers.

## Tests Performed:

- Tested ingame

## How to Test the Changes:

1. `.go xyz -8205.5 1679 -30 531 6.12`
2. pull the drones, see if most are pulled in packs of 5. Note: there are several unlinked spawns, which is intentional
3. kill some Drones, see if they despawn in 5s and respawn in 25s
4. kill Fankriss, see if **most** Drones and all Soldiers do not respawn

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1661014807166443400.sql [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+(87904, 87911, 0),
+(87905, 87911, 0),
+(87906, 87911, 0);
+
+-- Correct respawn timers & despawn
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 15300;
+
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 15300) AND (`source_type` = 0) AND (`id` IN (0));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(15300, 0, 0, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 41, 5000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, '');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1661014807166443400.sql:342]: comment

Comment (Gultask) [reply data/sql/updates/pending_db_world/rev_1661014807166443400.sql]: thanks, forgot about that.

**Comment by Maelthyrr** [comment]
Tested work but when Fankriss is dead, few drones respawn: 87827, 87766, 87746, 87744, 87701, 87724, 87723, 87701, 87768, 87798, 87771,  87774, 87787, 87804, 87825

**Comment by Gultask** [comment]
That is intended, yes. As long as they aren't in a linked formation they're supposed to respawn :)
They are the ones with the larger wander_distance radii and the ones with waypoints.

**Review by Nyeriah — APPROVED** [review]

**Pull request merged**